pkg/model/api/v1/datasource/http.go

```diff
@@ -70,43 +71,74 @@ func (h *HTTPAccess) validate() error {
 }
 
 type HTTPAllowedEndpoint struct {
+	Endpoint *regexp.Regexp `json:"endpoint" yaml:"endpoint"`
```

Comment:
I'd feel renaming this to `EndpointPattern` or something like that would be clearer now with this change?

Reply:
ah yes, good idea!